src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/api-authorization/authorize.service.ts

```diff
@@ -322,6 +322,11 @@ export class AuthorizeService {
       prefix: ApplicationName
     });
     this.userManager = new UserManager(settings);
+
+    this.userManager.events.addUserSignedOut(async () => {
+      await this.userManager.removeUser();
```

Comment:
This makes it so that if you log out from Identity you get logged out from the SPA too.